src/main/java/dev/suvera/keycloak/scim2/storage/events/ScimEventListener.java

```diff
@@ -67,16 +67,17 @@ private void handleUserEvent(AdminEvent event) {
 
             JsonNode representationJson = readJsonString(event.getRepresentation());
 
+            String userId = event.getResourcePath().split("/")[1];
+
             if (representationJson != null) {
                 JsonNode usernameNode = representationJson.get("username");
                 JsonNode federationLinkNode = representationJson.get("federationLink");
-                JsonNode idNode = representationJson.get("id");
```

Comment:
If this variable is not needed anymore, we can remove it.

Reply:
It is removed 😀